src/ui/index.css

```diff
@@ -0,0 +1,117 @@
+body, html {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, sans-serif; 
+}
```

Comment:
We should use normalize.css as a css reset. https://necolas.github.io/normalize.css/